app/serializers/disability_claim_detail_serializer.rb

```diff
@@ -34,6 +34,28 @@ def phase
     phase_from_keys 'claimPhaseDates', 'latestPhaseType'
   end
 
+  def tracked_items
+    items = []
+    items_src = object.data['consolidatedTrackedItemsList'] || []
```

Comment:
The data we have for this field in the CI env does not exactly match what I would expect (all tracked items for a claim in a single list). It looks to be only the tracked items that are still needed, which almost fits with that we need, but does not distinguish between what is needed from the veteran vs. what is needed from others. The status attribute is 'NEEDED' for both cases (vs. 'ACCEPTED' for items that were received). I think we may want to construct two lists for the UI, documents that are needed from the veteran (using `claimTrackedItems['stillNeededFromYouList']`) and documents that have been received (using `claimTrackedItems['receivedFromOthersList']` and `claimTrackedItems['receivedFromYouList']`)